web-server/v0.4/README.md

```diff
@@ -63,16 +68,16 @@ This will automatically open the application on [http://localhost:8000](http://l
 
 Both the production and development builds of the dashboard require specific configurations in order to run on their respective environment.
 
-Create and include a JSON config file in the root directory. Please reference the following example for required configuration fields.
-
-`dev.config.json`
+Modify the `datastoreConfig.js` file in the mock directory. Please reference the following example for required configuration fields.
```

Comment:
This is likely going to cause folks a problem, because a person can easily check-in a fully populated `datastoreConfig.js` file when they shouldn't.

Can we provide just a `datastoreConfig.js.example` which we tell people to copy to `datastoreConfig.js` where we add an explicit entry to ignore that file in `.gitignore`?

Reply:
Yes! This would be a much better approach.